src/main/java/frc/robot/commands/MoveBetweenBars.java

```diff
@@ -0,0 +1,20 @@
+package frc.robot.commands;
+
+import com.spikes2212.command.genericsubsystem.commands.MoveGenericSubsystem;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.subsystems.ClimberWinch;
+
+/**
+ * Moves the dynamic climbers downward towards the ground until they hit the third bar.
+ */
+public class MoveBetweenBars extends SequentialCommandGroup {
```

Comment:
Wow, this is a great piece of code! Perhaps, you should use it in your code. At least that's what I would do if I were you.

Reply:
damnnnnn